packages/UI/src/components/organisms/PositionCard/PositionCard.tsx

```diff
@@ -0,0 +1,178 @@
+
+const content = {
+  title: 'Web Developer',
+  level: 'junior',
+  timeCommittment: '5 hrs per week',
+  role: ' We are looking for a junior web developer who is willing and eagerto learn various web technologies mainly in the field of React while working within a team of ambitious developers! Our goal is to designa platform that brings people of from a diverse range of specialties together to collaboratively design the best possible software solutions. This project provides a space for people to establish and develop their ideas into working projects with input from others.',
+  tags: ['unity', 'react', 'social', 'agile', 'digital media'],
+  benefits:
+    " Be proficient in React.js and/or Next.js (strongly prefer both or atleast Next) Experience with GitHub and version control using Git Backend experience preferred, especially using a CMS Be able tocommit at least 5 hours a week Self-sufficient learner Communicate with us on a weekly basis to see the progress you've been making",
+  expecations:
+    "  Be proficient in React.js and/or Next.js (strongly prefer both or at least Next) Experience with GitHub and version control using Git Backend experience preferred, especially using a CMS Be able to commit at least 5 hours a week Self-sufficient learner Communicate with us on a weekly basis to see the progress you've been making",
+};
+
+const TagsContainer = ({ list }: string[]) => {
+  return (
+    <Box flexDirection="column" gap="16px" padding="32px 32px 16px 8px">
+      <Typography type="h5">Position tags</Typography>
+      <Box flexWrap="wrap" gap="8px">
+        {list.map((tag, i) => (
+          <Tags
+            key={`tag-${i}`}
+            bgColor="GREYSCALE_BLACK"
+            onClose={() => {}}
+            txtColor="GREYSCALE_WHITE"
+          >
+            {tag}
+          </Tags>
+        ))}
+      </Box>
+    </Box>
+  );
+};
+
+const CardDescription = ({ title, subtitle, body }) => {
+  return (
+    <Box
+      padding="32px 8px 16px"
+      flexDirection="column"
+      gap="16px"
+      maxWidth="450px"
+    >
+      <Typography type="h3">{title}</Typography>
+      <Typography type="h5">{subtitle}</Typography>
+      <Typography type="p">{body}</Typography>
+    </Box>
+  );
+};
+
+const PositionCard = ({
+  interaction,
+  status,
+  buttonStyle,
+  style,
+}: PositionCardProps) => {
+  const [open, setOpen] = useState(status);
+  const [height, setHeight] = useState(0);
+  const ref = useRef<HTMLDivElement>(null);
+
+  useLayoutEffect(() => {
+    const height = ref?.current?.clientHeight;
+    if (!ref) return;
+    ref.current.style.height = height + 'px';
+    setHeight((prev) => height || 0);
+    console.log(height);
+  }, [height]);
+
+  return (
+    <Container>
+      <Box gap="1rem">
+        <Box
+          style={{
+            background: 'black',
+            color: 'white',
+            borderRadius: '8px',
+            position: 'relative',
+            flexShrink: 0,
+          }}
```

Comment:
@Nouri-Anouar  looks like a Layer to me. We have to wait on [@alston-hsu](https://github.com/alston-hsu) to implement the rest of the layer styles and then use it here.

Reply:
Okay sounds good. 
I thought we already have the Layer component:
![image](https://user-images.githubusercontent.com/79311912/210221852-11088944-f2c7-47df-b559-f9c8b47d9499.png)

Reply:
@Nouri-Anouar we do have it but we don't have this specific one with that borderRadius. it's currently WIP